kubernetes/windows/main.ps1

```diff
@@ -357,8 +415,14 @@ function Set-EnvironmentVariables {
     else {
         Write-Host "Failed to set environment variable KUBERNETES_PORT_443_TCP_PORT for target 'machine' since it is either null or empty"
     }
+
+    if (![string]::IsNullOrEmpty($isAADMSIAuth) -and $isAADMSIAuth.ToLower() -eq 'true') {
+        Set-AMAEnvironmentVariables
```

Comment:
This can break 1P integration. lets chat how to handle this to ensure all the scenarios works as expected.